src/config.json

```diff
@@ -1,21 +1,18 @@
 {
-  "api": {
-    "host": "default"
-  },
-  "resources": {
-    "host_thumbnails": "http://localhost:4567/thumbnails/:s",
-    "host_objects": "http://localhost:4567/objects/:o"
-  },
   "competition": {
```

Comment:
Sensible defaults for DRES are debatable

Reply:
As long as the host is not enabled, we're good i think